components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,111 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025, Carsten Grzemba. All rights reserved.
+#
+
+# abreviation SRSS is used for SunRay Server Software
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# graphical desktop
+depend fmri=pkg:/mate_install type=require
+
+# DHCP service
+depend fmri=pkg:/service/network/dhcp/isc-dhcp type=require
+# empty package only for satisfy installation requirements
+# SRSS on Solaris uses the old Sun DHCP server but on OI we use ISC-DHCP
+depend fmri=pkg:/service/network/dhcp type=require
+
+# SUNWutkio-libs package dependency, empty packages only
+# for satisfy installation requirements
+depend fmri=pkg:/consolidation/X/X-incorporation type=require
+depend fmri=pkg:/compatibility/packages/SUNWxwplt type=require
+
+# for Sun JRE, needed for service auth.utauthd.utauthd,
+# additional native libraries of SRSS is used
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libCrun.so.1
+
+# SRSS configure requirements rpm2cpio,
+# see https://docs.openindiana.org/handbook/sunray/#configure-isc-dhcp
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/rpm2cpio
+
+# in OI legacy/retired 64bit packages GDM and Gnome greeter,
+# only gdm-session-worker has to run in 32-bit, it needs SRSS libraries only available in 32-bit
+#
+# pkg.debug.depend.file does not working yet, there is no appropriate package in the OI repo
+# hence the kind of definition with named packages
+#
+depend fmri=pkg:/system/display-manager/gdm-2 type=require
+depend fmri=pkg:/gnome/gnome-session-2 type=require
+depend fmri=pkg:/library/desktop/gnome-desktop-2 type=require
```

Comment:
Why do you need `pkg:/` prefixes here?
Why are these packages needed at all (some good explanation is missing)?